umbra-js/src/utils/utils.ts

```diff
@@ -350,6 +350,10 @@ export async function getEthSweepGasInfo(
     provider.getCode(to),
     provider.getNetwork(),
     provider.getBalance(from),
+    // getGasPrice is officially deprecated in favor of getFeeData, as the
+    // former uses pre-1559 gas pricing. But we use it anyway because it makes
+    // it easier to calculate a price to sweep the account (1599 factors in
+    // refunds that we don't care about).
```

Comment:
I think it's ok to delete this comment, or just simplify it to say "We use `getGasPrice` instead of `getFeeData` because we don't care about refunds here, so using type 0 transactions makes it easier to calculate a price to sweep the account, and 1559 is not supported on some chains". Mainly just commenting that type 0/1 transactions (EIP-1559 txs are type 2) aren't technically deprecated, and are still a valid alternative (just for most people, 1559 is better). Also some networks we support might not support `getFeData`/1559, e.g. I don't think Arbitrum does nor Optimisim (currently, Bedrock will though)

Reply:
Resolving. This comment was updated accordingly